torch/nn/functional.py

```diff
@@ -670,6 +670,10 @@ def binary_cross_entropy(input, target, weight=None, size_average=True):
                 sizeAverage is set to False, the losses are instead summed
                 for each minibatch.
     """
+    if not target.is_same_size(input):
+        warnings.warn("Using a target size ({}) that is different to the input size ({}) is deprecated. "
```

Comment:
Just a question: won't this function raise the warnings twice, one in the functional interface, and the other in the `_functions`?